duet/views/funnel_ios_new.view.lkml

```diff
@@ -0,0 +1,429 @@
+view: funnel_ios_new {
+
+    derived_table: {
+      sql: WITH views_data AS (
```

Comment:
The old part of the query should be replaced with the new table.
It'd look like this:
```
WITH
  retention AS (
  SELECT
    client_id,
    first_seen_date,
    country,
    COALESCE(MAX(IF (submission_date = DATE_ADD(first_run_date, INTERVAL 27 DAY), mozfun.bits28.active_in_range(days_seen_bits, -6, 7), NULL) ), FALSE) AS week_4_retention,
    COUNT(DISTINCT CASE WHEN DATE_DIFF(submission_date, first_run_date, day) >= 1 AND DATE_DIFF(submission_date, first_run_date, day) <= 28 AND days_since_seen = 0 THEN submission_date END) AS first_28_days_active_days
  FROM
    `mozdata.firefox_ios.baseline_clients_last_seen`
  WHERE
    submission_date >= '2022-01-01'
    AND first_seen_date >= '2022-01-01'
    AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01') -- incident filter
  GROUP BY
    1,
    2,
    3 ),
  ret_aggs AS (
  SELECT
    first_seen_date AS date,
    country,
    COUNT(DISTINCT CASE WHEN first_28_days_active_days > 1 THEN client_id END) multi_days_users,
    COUNT(DISTINCT CASE WHEN week_4_retention = TRUE THEN client_id END) week_4_retained
  FROM
    retention
  GROUP BY
    1,
    2 )
SELECT
  date,
  country,
  views,
  total_downloads AS downloads,
  new_profiles,
  activations,
  multi_days_users,
  week_4_retained
FROM
  `moz-fx-data-shared-prod.firefox_ios.app_store_funnel`
JOIN ret_aggs
USING (date, country)
```